sdk/aqueduct/integrations/connect_config.py

```diff
@@ -147,18 +147,37 @@ class SlackConfig(BaseConnectionConfig):
     enabled: bool
 
 
-class AWSConfig(BaseConnectionConfig):
-    access_key_id: str
-    secret_access_key: str
-
-
 class _SlackConfigWithStringField(BaseConnectionConfig):
     token: str
     channels_serialized: str
     level: str
     enabled: str
 
 
+class DynamicK8sConfig(BaseConnectionConfig):
+    keepalive: Optional[str]
+    cpu_node_type: Optional[str]
+    gpu_node_type: Optional[str]
+    min_cpu_node: Optional[str]
+    max_cpu_node: Optional[str]
+    min_gpu_node: Optional[str]
+    max_gpu_node: Optional[str]
```

Comment:
Could you add some description for each of these? Specifically wondering about things like:
- Are these strings of numbers for the min/max stuff? 
- What does do the min/max mean? Are they hard bounds? Are they inclusive? What size is the cluster initially created to, and can that be outside of the bounds?
- What are the accepted `cpu/gpu_node_type`s?

Reply:
Added description to each field. In terms of the accepted cpu/gpu node types, it's whatever that's supported by AWS that the account has permission to provision.